VideoApi/VideoApi/Validations/UpdateParticipantRequestValidation.cs

```diff
@@ -5,16 +5,10 @@ namespace VideoApi.Validations
 {
     public class UpdateParticipantRequestValidation : AbstractValidator<UpdateParticipantRequest>
     {
-        public static readonly string NoNameErrorMessage = "Fullname is required";
```

Comment:
how come we're removing these rules? If we need to do a partial update, maybe we should have a new `UpdateDisplayName` request, or remove the properties

Reply:
So im gonna spin up another ticket after this to remove all this fields from the database anyway and therfore the properties from the request contracts. I needed to remove the validation in this pr though, as the videoweb makes a request to this endpoint to update the display name. As the request it makes no longer has these details anymore, it returns a 400 bad response.